dwave/cloud/client.py

```diff
@@ -675,17 +718,28 @@ def solvers(self, name=None, qpu=None, software=None, vfyc=None,
             class.
 
         Examples:
-            Get a solver not based on VFYC, with 2048 qubits::
+            Get all solvers based on VFYC, with 2048 qubits::
+
+                solver = client.solvers(vfyc=True, num_qubits=2048)
+                solver = client.solvers(vfyc__eq=True, num_qubits__eq=2048)
```

Comment:
We'll get support questions on why two code lines are needed. I suggest instead one line with mixed syntax:
solver = client.solvers(vfyc=True, num_qubits__eq=2048)

Reply:
Just wanted to emphasize either works. I'll add a "or".